docs/doc.md

```diff
@@ -1,26 +1,29 @@
 
 # Motivation
-The Android Canvas API is a powerful tool, but sometimes it can be very hard to use. This usually result in a lot of boilerplate, which makes the code harder to read and debug.
+The Android Canvas API is a powerful tool, but sometimes can be very hard to use. This usually results in a lot of boilerplate, which makes the code harder to read and debug.
 
-Using newer language features provided by Kotlin, this library intend to make Android's Canvas as easy to use as possible.
+Using newer language features provided by Kotlin, this library is intended to make Android's Canvas as easy to use as possible.
 
 
 # Modularisation
 Enamel is a set of libraries consisting of multiple modules:
 
+
+//TODO:  `Plug for the *Geometry*` - is this meant to say plugin?
+
 - Core (Kotlin): Basic mathematical functions used by other modules.
-- Geometry (Kotlin): Geometry shapes and concepts such a point, angle, rectangle, circle, etc.
-- Geometry (Android): Plug for the *Geometry* module into Android, it contains extensions on Canvas related classes such as **Canvas**, **Path**, **View**.
-- Touch (Android): Utility functions to handle touch events in an easy manner.
+- Geometry (Kotlin): Geometric shapes and concepts - e.g. point, angle, rectangle, circle, etc.
+- Geometry (Android): Plug for the *Geometry* module into Android, it contains extensions for Canvas related classes such as **Canvas**, **Path**, **View**.
```

Comment:
is this meant to say 'plugin' rather than 'plug'?

Reply:
Yes